src/main/java/org/openhab/binding/volumio2/handler/Volumio2Handler.java

```diff
@@ -193,7 +210,7 @@ private void handlePlaybackCommands(Command command) {
                     volumio.play();
                     break;
                 case PAUSE:
-                    volumio.pause();
+                    volumio.stop();
```

Comment:
I guess this is done because of Radio streams.. but how does it differ from pause when you're playing local music?

We could just add an additional channel stop.

Reply:
Hi Patrick, you're right, I've done it because of the radio streams I want to control with my snips. I'll add a new stop channel.